Kudu.Services.Web/Content/Scripts/FileBrowser.js

```diff
@@ -80,39 +80,47 @@ function showAceHelpModal() {
     });
 }
 
-// File upload progress indicator
-var copyProgressObjects = {};
+
 var copyObjectsManager = {
     init: function() {
-        copyProgressObjects = {};
+        this._copyProgressObjects = {};
+        this.infoMessage = '';
+    },
+    getInfoMessage: function() {
+        return this._infoMessage;
+    },
+    setInfoMessage: function(message) {
+        this._infoMessage = message;
     },
-    addCopyStats: function (uri, loadedData, totalData
-   )  { 
-        if (copyProgressObjects[uri]) {
-            if (loadedData === totalData) { 
-                copyProgressObjects[uri].endDate = $.now();
+    addCopyStats: function (uri, loadedData, totalData) {
+
+        uri = uri.substring(uri.indexOf('/vfs')+5, uri.length); // slice uri to be prettier
```

Comment:
for "uri", will be good to add a comment to have a sample uri. make reading the code easier.